types/emscripten/index.d.ts

```diff
@@ -1,7 +1,8 @@
-// Type definitions for Emscripten
+// Type definitions for Emscripten 1.38.33
```

Comment:
This appears to be a malformed header. Could you remove the line in the `tslint.json` file in this folder that says `dt-header: false` and fix any errors that come up? You will want this first line to say `Type definitions for Emscripten 1.38`. You are welcome to remove any other such line from the `tslint.json` file, though that might be a lot of work depending on what issues are being ignored. Thanks!